keyboards/keebio/iris/keymaps/johnstegeman/keymap.c

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2023 John Stegeman
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#include "print.h"
+#include "light_layers.h"
+
+enum layers {
+_COLEMAK,
+_NAVNUM,
+_CONTROL,
+};
+
+// Enter when tapped, shift when held
+#define SHEN MT(MOD_LSFT, KC_ENT) 
+
+// Enter when tapped, alt when held
+#define ALEN MT(MOD_LALT, KC_ENT) 
+
+#define CTX LT(0,KC_X)
+#define CPC LT(0,KC_C)
+#define PSV LT(0,KC_V)
+
+// Space when tapped, shift when held 
+#define SHSP MT(MOD_RSFT, KC_SPC)
+
+enum custom_keycodes {
+  COLEMAK = QK_USER,   
+};
+
```

Comment:
You're not using this at all, so you could safely remove this.
```suggestion
```

Reply:
done